scripts/setupwrm.ps1

```diff
@@ -4,7 +4,7 @@ write-output "Running User Data Script"
 write-host "(host) Running User Data Script"
 
 # Create username and password
-wmic useraccount where "name='Administrator'" set PasswordExpires=FALSE
+wmic useraccount where "name='dqsupport'" set PasswordExpires=FALSE
```

Comment:
I am concerned that by doing this the Administrator account on the Windows boxes will expire - this might need to be used to reset regular user passwords.

Reply:
This is a requirement from ITHC to use a different administrator account that the default account.